src/components/Edit/ChangeSummary.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <v-card flat id="change-summary">
+    <!-- Section Header -->
+    <div class="summary-header px-4 mb-2 rounded-t">
+      <v-row no-gutters>
+        <v-col cols="9">
+          <v-icon color="primary">mdi-file-document-edit-outline</v-icon>
+          <label class="summary-title">Summary of Changes to File</label>
+        </v-col>
+
+        <!-- Actions -->
+        <v-col cols="3" class="mt-n2">
+          <div class="actions mr-4">
+            <v-btn
+              text color="primary"
+              id="btn-change"
+              :disabled="isBusySaving"
+              @click="onChangeClicked()"
+            >
+              <v-icon small>mdi-pencil</v-icon>
+              <span>Change</span>
+            </v-btn>
+            <v-btn
+              text color="primary"
+              id="btn-delete"
+              :disabled="isBusySaving"
+              @click="onDeleteClicked()"
+            >
+              <v-icon small>mdi-delete</v-icon>
+              <span>Delete</span>
+            </v-btn>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+
+    <!-- Business Name -->
+    <template v-if="hasBusinessNameChanged">
+      <v-divider class="mx-4" />
+      <div class="section-container business-name-summary">
+        <v-row no-gutters>
+          <v-col cols="3">
+            <label><strong>Company Name</strong></label>
+          </v-col>
+
+          <v-col cols="8" class="mt-n1">
+            <div class="company-name font-weight-bold text-uppercase">{{ companyName }}</div>
+            <div class="company-name mt-2">{{ getNameRequest.nrNumber }}</div>
+          </v-col>
+        </v-row>
+      </div>
+    </template>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF } from '@/interfaces'
+import { DateMixin, EnumMixin, FilingTemplateMixin, LegalApiMixin, PayApiMixin } from '@/mixins'
+
+@Component({
+  components: {}
+})
+export default class ChangeSummary extends Mixins(
+  DateMixin,
+  EnumMixin,
+  FilingTemplateMixin,
+  LegalApiMixin,
+  PayApiMixin
+) {
+  // Global getters
+  @Getter getApprovedName!: string
+  @Getter getBusinessNumber!: string
+  @Getter isBusySaving!: boolean
+
+  // Change flag getters
+  @Getter hasBusinessNameChanged!: boolean
+
+  // Global actions
+  @Action setSummaryMode!: ActionBindingIF
```

Comment:
Depending on how "low level" this component is, it might be better for it to NOT use the store (use props and events instead).

I would say...
1. top level - the main view - should not know too much detail abouts its components but can use the store, talk with App.vue, fetch data
2. middle level - a simple component on the main page/view - doesn't need to know about top level stuff, provides interface between its low-level sub-components and store
2b. middle level - if middle component is complex, it may contain middle level components of its own
3. low level - does the detailed work/logic - ideally should have minimal dependencies, for example should not get/set from/to store; this low-level is an easy candidate for reuse/sharing

Reply:
While i do agree this component in particular is a reflection of store data for the most part, so i think if perhaps any layer to is to have access to store, it makes sense here.
Either we bind the getters here or the alternative is bind them on the parent, pass them into the component and create the prop bindings etc. 
The latter just feels like more code, with more potential points of failure. I'm all for it, just hard to see the value in this case in particular.

Reply:
Since this component's parent is a view, I agree that it's fine that THIS component handle all the store stuff.